diff --git a/Makefile b/Makefile
index d8f09a8..38da624 100644
--- a/Makefile
+++ b/Makefile
@@ -3,7 +3,7 @@ CLOSURE=$(CLOSURE_DIR)/compiler.jar
 BROWSER=chromium
 NASM_TEST_DIR=./tests/nasm
 
-all: build/v86_all.js
+all: build/v86_all.js build/libv86.js
 browser: build/v86_all.js
 
 ACPI=false
@@ -122,7 +122,8 @@ build/libv86.js: $(CLOSURE) src/*.js lib/*.js src/browser/*.js
 		$(CLOSURE_FLAGS)\
 		--compilation_level SIMPLE\
 		$(TRANSPILE_ES6_FLAGS)\
-		--output_wrapper ';(function(){%output%}).call(this);'\
+		--language_out NO_TRANSPILE\
+		--output_wrapper '%output%;delete window.V86;delete window.V86Starter;export {V86Starter as default};'\
 		--js $(CORE_FILES)\
 		--js $(BROWSER_FILES)\
 		--js $(LIB_FILES)
